examples/website/safegraph/app.js

```diff
@@ -48,26 +52,27 @@ export function renderToDOM(container, data) {
       }
     });
 
-    renderLayers(map, data);
+    renderLayers(map, deckOverlay, data);
   });
 
+  map.addControl(deckOverlay)
+
   return {
-    update: newData => renderLayers(map, newData),
+    update: newData => renderLayers(map, deckOverlay, newData),
     remove: () => {
       map.remove();
     }
   };
 }
 
-function renderLayers(map, data) {
+function renderLayers(map, deckOverlay, data) {
```

Comment:
Is `map` still needed in this function?

Reply:
It's used for `beforeId: getFirstLabelLayerId(map.getStyle())`